modin/engines/ray/utils.py

```diff
@@ -164,3 +165,6 @@ def initialize_ray(
         )
 
         ray.worker.global_worker.run_function_on_all_workers(_import_pandas)
+
+    num_cpus = int(ray.cluster_resources()["CPU"])
+    NPartitions.put(num_cpus)
```

Comment:
What if `MODIN_CPUS` was be defined?

Reply:
In cluster case `CpuCount` isn't equal to actual used cores, is it?

Reply:
Yes, it is. In cluster case `CpuCount` would be equal to number of cpus on a head node.

Reply:
This is contrary to her definition - `How may CPU cores to utilize across the whole distribution`

Reply:
We could update `CpuCount` to actual number of cpus after engine initialization. Then, a user will be able to use `CpuCount.get()` to get the number.

Reply:
However, it is confusing a little because a user can set `MODIN_CPUS` at first working on a cluster and then call `CpuCount.get()` after engine initialization. The numbers will be different.

Reply:
I think, correcting the definition of `CpuCount` will be enough. Then, a user will be able to compute the number of cpus myself via `CpuCount.get() * num_nodes`.